### DISCUSSION TITLE
How can I fix my API to not look like a tracker?

### DISCUSSION THREAD

**Discussion by felipe-chamas** [EFForg/privacybadger#2731] [Q&A]
Sometimes my AWS ApiGateway requests get blocked on JS by the extension, how can I make sure it doesn't happen to my clients using the extension?
I mean, what can I do to my API so that it does not look like a tracker?

**Comment by ghostwords** [comment]
Hello! Please take a look at ["How do I stop Privacy Badger from blocking me?"](https://privacybadger.org/#-I-am-an-online-advertising-tracking-company.--How-do-I-stop-Privacy-Badger-from-blocking-me) on Privacy Badger's homepage.

I could look into what exactly happens in your case if you shared more details about how I might go about reproducing your problem. It will help to provide debugging information by following the [instructions on our Wiki](https://github.com/EFForg/privacybadger/wiki/Find-out-why-Privacy-Badger-is-blocking-a-domain).
